Makefile

```diff
@@ -295,6 +298,10 @@ test-check-containers:
 tidy: tidy-js
 	tools/tidy
 
+.PHONY: test-containers-compose
+test-containers-compose:
+	for i in $(wildcard container/**/docker-compose.yaml); do cd "$$(echo $$i | cut -d "/" -f-2)"; sudo docker-compose up -d || exit 1; cd ../../; done
```

Comment:
```suggestion
	for i in $(dir $(wildcard container/**/docker-compose.yaml)); do (cd $$i; sudo docker-compose up -d || exit 1); done
```

Reply:
Not sure this works as expected though. I think this will only exit the subshell but not stop the execution of the loop.

`(cd $$i; sudo docker-compose up -d || exit 1) || exit 1` might work though

Reply:
ah right. this should do it:
```
do (cd $$i; sudo docker-compose up -d) || exit 1
```

Reply:
Thanks, seems work perfectly